doc/Language/classtut.pod6

```diff
@@ -199,87 +101,61 @@ declarations. The example code includes attributes (state), introduced
 through the C<has> keyword, and behaviors, introduced through the C<method>
 keyword.
 
-X<|type object>
-X<|DEFINITE>
-X<|.DEFINITE>
-
-Declaring a class creates a new I<type object> which, by default, is installed
-into the current package (just like a variable declared with C<our> scope).
-This type object is an "empty instance" of the class. For example, types such as
-C<Int> and C<Str> refer to the type object of one of the Raku built-in
-classes. The example above uses the class name C<Task> so that other code can
-refer to it later, such as to create class instances by calling the C<new>
-method.
-
-You can use the C<.DEFINITE> method to find out if what you have is an instance
-or a type object:
-
-=begin code
-say Int.DEFINITE; # OUTPUT: «False␤» (type object)
-say 426.DEFINITE; # OUTPUT: «True␤»  (instance)
-
-class Foo {};
-say Foo.DEFINITE;     # OUTPUT: «False␤» (type object)
-say Foo.new.DEFINITE; # OUTPUT: «True␤»  (instance)
-=end code
-
-You can also use type "smileys" to only accept instances or type objects:
-
-=begin code
-multi foo (Int:U) { "It's a type object!" }
-multi foo (Int:D) { "It's an instance!"   }
-say foo Int; # OUTPUT: «It's a type object!␤»
-say foo 42;  # OUTPUT: «It's an instance!␤»
-=end code
-
```

Comment:
Moved. See the corresponding comment and https://github.com/Raku/doc/commit/d78116f76e5ed801616eeadaee5be8366a55dbf6.